pkg/cloud/aws/services/cloudformation/bootstrap.go

```diff
@@ -203,6 +193,77 @@ func controllersPolicy(accountID string) *iam.PolicyDocument {
 					"StringLike": map[string]string{"iam:AWSServiceName": "elasticloadbalancing.amazonaws.com"},
 				},
 			},
+			{
+				Effect:   iam.EffectAllow,
+				Resource: iam.Resources{"*"},
+				Action: iam.Actions{
+					"ec2:CreateTags",
```

Comment:
Will this limit us from being able to apply the AddtionalTags specified in the ProviderConfig: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/64079bfbf8278026a5fa3cc51b295f4fb1e2d248/pkg/apis/awsprovider/v1alpha1/awsmachineproviderconfig_types.go#L48

Reply:
Have revised the policy to have `ForAnyValue`, so these are minimum requirements, not exact.